social_core/backends/auth0.py

```diff
@@ -21,6 +21,10 @@ def api_path(self, path=''):
         return 'https://{domain}/{path}'.format(domain=self.setting('DOMAIN'),
                                                 path=path)
 
+    def auth_extra_arguments(self):
+        """Return extra arguments needed on request-token process"""
+        return self.setting('AUTH_EXTRA_ARGUMENTS', {})
```

Comment:
Why is this function needed? The default defined in the base class will do the same.

Reply:
Ouch, you are right. I didn't see that in base class ... sorry for that!